src/main/java/frc/robot/RobotContainer.java

```diff
@@ -161,6 +164,10 @@ public RobotContainer() {
   }
 
   private void configureAuto() {
+
+    NamedCommands.registerCommand("runIntake", runIntake);
+    NamedCommands.registerCommand("aimAndFire", aimAndFire);
```

Comment:
oh my god this is so gross and I hate it with all of my being but if it has to be this way okay

Reply:
Why do you hate this?